recipe/meta.yaml

```
@@ -93,8 +93,9 @@ outputs:
         - python
         - {{ pin_subpackage('numpy-base', exact=True) }}
         # openblas or mkl runtime included with run_exports
-        - mkl_fft  # [blas_impl == 'mkl']
-        - mkl_random # [blas_impl == 'mkl' and (not win or vc>=14)]
+        # current mkl_fft and mkl_random are not buildable for python 3.12
+        - mkl_fft  # [blas_impl == 'mkl' and (py<312)]
+        - mkl_random # [blas_impl == 'mkl' and (not win or vc>=14) and (py<312)]
```

```suggestion
        - mkl_fft  # [blas_impl == 'mkl' and py<312]
        - mkl_random # [blas_impl == 'mkl' and (not win or vc>=14) and py<312]
```
Redundant parentheses